adapters/oidc/js/pom.xml

```diff
@@ -18,6 +18,10 @@
         <js.projectDir>../../../js</js.projectDir>
         <js.adapter.projectDir>${js.projectDir}/libs/keycloak-js</js.adapter.projectDir>
         <js.adapter.distDir>${js.adapter.projectDir}/dist</js.adapter.distDir>
+        <!-- as the frontend-maven-plugin depends on workingDirectory, list all parts which should be ignored from the checksum -->
+        <maven.build.cache.exclude.1>${maven.multiModuleProjectDirectory}/js/libs/keycloak-js/node_modules</maven.build.cache.exclude.1>
```

Comment:
As far as I understand it `maven.multiModuleProjectDirectory` is not a supported variable, as such I could imagine this could change what value it has resulting in this exclude being not efficient sometimes, but not sure when as the variable is not documented

Reply:
I previously attempted to use `${project.basedir}` and a relative path, but that didn't work as the extension doesn't normalize paths. I'd like to go with this approach, and will create a PR for upstream for the extension to normalize paths.

Reply:
Sounds good